Fix color package

mtelvers opened the pull request:
Followup to https://github.com/ocaml/opam-repository/pull/26927

shonfeder committed:
Fix color package

See https://github.com/ocaml/opam-repository/issues/26956

shonfeder commented on packages/color/color.0.3.0/opam:
```diff
@@ -0,0 +1,43 @@
+  "ocamlformat" {= "0.26.2" & with-dev-setup}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
```suggestion
  ["dune" "subst"] {dev}
```

shonfeder committed:
Update packages/color/color.0.3.0/opam

ElectreAAS commented:
For the record here: the problem with #26927 was that line `url: {` instead of `url {`, is that correct?

shonfeder commented:
That's a funky quirk of the opam format.

shonfeder committed:
Add upper bound on color dependency

color 0.3.0 renamed the Rgba module to Rgb:
https://github.com/anuragsoni/color/blob/main/CHANGES.md#030

mseri commented:
Thanks!
